packages/builder/src/utils/eslint-utils.ts

```diff
@@ -22,10 +22,11 @@ export async function lint(
   const eslint = new projectESLint.ESLint({
     useEslintrc: true,
     overrideConfigFile: eslintConfigPath,
-    ignorePath: options.ignorePath || undefined,
-    fix: !!options.fix,
-    cache: !!options.cache,
-    cacheLocation: options.cacheLocation || undefined,
+    ignorePath: options.ignorePath ?? undefined,
```

Comment:
Why change these to use `??` instead of `||`?

`||` was already doing everything we needed it to, and now technically with your change `0`, `NaN` and `''` will be set preferentially to `undefined` for these values, which is not desired. It may be unlikely that those values make their way through the code paths here, but there needs to be an objective reason to make this change in the first place IMO, and it seems objectively less desirable to me.

Reply:
You're right. I'll revert this 😃